src/_pytest/config/__init__.py

```diff
@@ -238,6 +240,19 @@ def __init__(self):
         # Used to know when we are importing conftests after the pytest_configure stage
         self._configured = False
 
+    def is_blocked(self, name):
```

Comment:
It will be a bummer to write one, but I guess we should add a functional test for this?

Reply:
Yeah, tests are fine.. :D